src/tds/codec/token/token_env_change.rs

```diff
@@ -35,8 +35,8 @@ uint_enum! {
 }
 
 pub struct CollationInfo {
-    lcid_encoding: Option<&'static dyn Encoding>,
-    sortid_encoding: Option<&'static dyn Encoding>,
+    lcid_encoding: Option<&'static (dyn Encoding + Send + Sync)>,
```

Comment:
Same question here: should `Encoding: Send + Sync` ?

Reply:
It's not our trait though...

Reply:
ah! -- fair enough haha